Tweak/Core/Views/KayokoFavoritesTableView.m

```diff
@@ -47,23 +58,44 @@ - (UISwipeActionsConfiguration *)tableView:(UITableView *)tableView leadingSwipe
             completionHandler(YES);
         }];
         [translateAction setImage:[UIImage systemImageNamed:@"globe"]];
-        [translateAction setBackgroundColor:[UIColor systemBlueColor]];
+        // rgba(52, 152, 219, 1.0)
+        baseColorSeries = [UIColor colorWithRed:52.0/255.0 green:152.0/255.0 blue:219.0/255.0 alpha:1.0];
         [actions addObject:translateAction];
     }
 
+    if (baseColorSeries)
+    {
+        // Make color a bit different and set them to each action
+        for (NSUInteger i = 0; i < [actions count]; i++)
+        {
+            UIContextualAction *action = [actions objectAtIndex:i];
+            CGFloat hue, saturation, brightness, alpha;
+            [baseColorSeries getHue:&hue saturation:&saturation brightness:&brightness alpha:&alpha];
+            [action setBackgroundColor:[UIColor colorWithHue:hue saturation:saturation brightness:brightness - (i * 0.1) alpha:alpha]];
+        }
+    }
+
```

Comment:
Why I changed the color series you used before? You aligned blocks of different colors at one row, which will cause **optical illusion**.